modules/monitoring-limiting-scrape-samples-in-user-defined-projects.adoc

```diff
@@ -1,15 +1,23 @@
 // Module included in the following assemblies:
 //
-// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+// * monitoring/configuring-the-monitoring-stack.adoc
 
-[id="limiting-scrape-samples-in-user-defined-projects_{context}"]
-= Limiting scrape samples in user-defined projects
+//[id="limiting-scrape-samples-in-user-defined-projects_{context}"]
```

Comment:
Not critical, but this line could be removed.

Reply:
Thank you for highlighting this. I have now removed the line. I left it in as a reference because I wanted to check if any files had a reference to that ID once the new title had been reviewed and approved.